storage/store.go

```diff
@@ -229,7 +229,8 @@ type Store struct {
 	raft        raftInterface
 	closer      chan struct{}
 
-	mu          sync.RWMutex     // Protects variables below...
+	mu sync.RWMutex // Protects variables below...
+	// TODO(Tobias) change int64 to uint64?
 	ranges      map[int64]*Range // Map of ranges by Raft ID
```

Comment:
That's what it started out as (used to be a 64 bit range ID). If we want to change Raft ID to a uint64 to match underlying raft implementation, that's fine, but it's definitely a separate change, as it threads through protos and a large amount of code.